api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java

```diff
@@ -408,7 +408,7 @@ private Map<String, ProtoFile> loadFilesWithLocations() {
         files.filter(p -> !Files.isDirectory(p) && p.toString().endsWith(".proto"))
             .forEach(path -> {
               // relative path will be used as "import" statement
-              String relativePath = baseLocation.relativize(path).toString();
+              String relativePath = baseLocation.relativize(path).toString().replace("\\", "/");
```

Comment:
Could we get rid of this and possibly replace it with some native java nio API calls if required?

Reply:
What are you looking to get rid of? The goal of this PR wasn't to rewrite this function.

Reply:
Yes, but "fixing tests on windows" doesn't imply altering the production code. I believe this could lead to unexpected side effects in production, perhaps on non-windows OS.

Reply:
That is fair. I can't say I understand this implementation well enough to know why one part uses the OS separator and another doesn't, which causes the mismatch. This part is using the OS separator, and I would wager rewriting it will not solve the problem. 

I can revert this portion if you like, but the E2E tests are passing on multiple OSes now and the slash being replaced is only used by Windows.

Reply:
I would prefer not to have any hacky solutions in production code to be able to run tests on some weird environments.
I believe this can be somehow solvable with standard APIs, but I don't have time to dig into this.
I'd be happy to accept a PR which can alter this production code but in a way which doesn't introduce OS-specific hacks. Otherwise, at this point, let's run the tests in wsl or docker until we don't have a better solution.